examples/drawing/plot_four_grids.py

```diff
@@ -3,27 +3,40 @@
 Four Grids
 ==========
 
-Draw a graph with matplotlib.
+Draw a 4x4 graph with matplotlib.
+
+This example illustrates the use of keyword arguments to `networkx.draw` to
+customize the visualization of a simple Graph comprising a 4x4 grid.
 """
 
 import matplotlib.pyplot as plt
 import networkx as nx
 
 G = nx.grid_2d_graph(4, 4)  # 4x4 grid
 
-pos = nx.spring_layout(G, iterations=100)
-
-plt.subplot(221)
-nx.draw(G, pos, font_size=8)
-
-plt.subplot(222)
-nx.draw(G, pos, node_color="k", node_size=0, with_labels=False)
-
-plt.subplot(223)
-nx.draw(G, pos, node_color="g", node_size=250, with_labels=False, width=6)
-
-plt.subplot(224)
+pos = nx.spring_layout(G, iterations=100, seed=39775)
+
+# Create a 2x2 subplot
+fig, all_axes = plt.subplots(2, 2)
+ax = all_axes.flat
+
+nx.draw(G, pos, ax=ax[0], font_size=8)
+nx.draw(G, pos, ax=ax[1], node_size=0, with_labels=False)
+nx.draw(
+    G,
+    pos,
+    ax=ax[2],
+    node_color="tab:green",
+    edge_color="tab:gray",
```

Comment:
Add
```
    edgecolors="tab:gray",
```

Reply:
Is this different from `edge_color="tab:gray"`? If I use "edgecolors" as the kwarg name instead it reverts back to black edges.

Reply:
Oh nvm - I see now, it's the node edgecolor